exercises/practice/rational-numbers/.docs/instructions.md

```diff
@@ -4,7 +4,7 @@ A rational number is defined as the quotient of two integers `a` and `b`, called
 
 ~~~~exercism/note
 Note that mathematically, the denominator can't be zero.
-However in many implementations of rational numbers, you will find that the denominator is allowed to be zero with behaviour similar to positive or negative infinity in floating point numbers.
+However, in many implementations of rational numbers, you will find that the denominator is allowed to be zero with behavior similar to positive or negative infinity in floating point numbers.
```

Comment:
`behaviour` mostly used in British English, but it seems to me that the text is more in keeping with the style of American English. Anyway, the final decision is up to maintainers :)